(DOCSP-10996) Added section to note how to connect to specific database

jocelyn-mendez opened the pull request:
Added section to note how to connect to specific database. 

jira: https://jira.mongodb.org/browse/DOCSP-10996

staging: https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-10996/connect

jocelyn-mendez commented:
Hey @jeff-allen-mongo, I updated this page. Let me know your thoughts when you get a chance, thanks!

jeff-allen-mongo commented on source/connect.txt:
```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
+
+     .. code-block:: sh
+
+         mongosh 'mongodb://localhost:27017/db1' 
```
This renders a bit weird (URI is italicized) I think because you're using single quotes `'` instead of double `"`

jeff-allen-mongo replied:
Can we add a sentence below this code block (but still in the example block) what this does? When we include code, we should try to provide some sort of context.

Something like: "The following connection string URI connects to the `db1` database".

jeff-allen-mongo commented on source/connect.txt:
```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
+
+- You can specify a database in the URL path when passing a connection
+  string. The database in the URL path is automatically selected unless
+  you specify a different database.
```
This last sentence is confusing to me. Reading this, my question is "specify a different database where?". Can you explain (in a comment here) the idea you want to get across?

jeff-allen-mongo commented on source/connect.txt:
```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
+To connect to a specific database:
+
+- You can specify a database in the URL path when passing a connection
+  string. The database in the URL path is automatically selected unless
```
It may be helpful to include a sentence here stating what the default database that mongosh connects to is (i.e., if I don't specify a database in my URI, what database do I connect to?)

jocelyn-mendez commented:
Hey @jeff-allen-mongo, I updated the page so feel free to take a look when you get a chance, thanks!

Updated staging: https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-10996/connect

jeff-allen-mongo commented on source/connect.txt:
```diff
@@ -204,6 +204,23 @@ For ``tls`` connections:
+To connect to a specific database, specify a database in your
+`connection string URI path
+<https://docs.mongodb.com/manual/reference/connection-string/>`__. If
+you do not specify a database in your URI path, you connect to a Test
```
[nit] Since this is case-sensitive, I think it's important to say a `test` database. Also, since "test database" can mean different things in different contexts, I think we should say: "you connect to a database named `test`".

Use double backticks (\`\`) around `test` to get the monospaced styling so it stands out.

jeff-allen-mongo commented on source/connect.txt:
```diff
@@ -204,6 +204,23 @@ For ``tls`` connections:
+
+   .. example::
+
+     The following connection string URI connects to database db1.
```
[nit] Let's also use the double backticks around `db1` to make it standout a bit more.

```
The following connection string URI connects to database ``db1``.
```
